src/main/kotlin/org/rust/cargo/project/CargoProjectResolver.kt

```diff
@@ -31,7 +31,8 @@ class CargoProjectResolver : ExternalSystemProjectResolver<CargoExecutionSetting
                                     settings: CargoExecutionSettings?,
                                     listener: ExternalSystemTaskNotificationListener): DataNode<ProjectData>? {
 
-        val pathToCargo = settings?.cargoPath ?: return null
+        settings ?: return null
```

Comment:
We have 14 instances of this pattern in our code base, I'll fix them and update styleguide